tests/unittests/CoreText/CTFontTests.mm

```diff
@@ -194,15 +195,16 @@ virtual void TearDown() {
 static const CFStringRef c_courierNewBoldName = CFSTR("Courier New Bold");
 static const CFStringRef c_courierNewBoldItalicName = CFSTR("Courier New Bold Italic");
 static const CFStringRef c_trebuchetMSItalicName = CFSTR("Trebuchet MS Italic");
+static const CFStringRef c_timesNewRomanName = CFSTR("Times New Roman");
```

Comment:
yup!

---
In reply to: [86892725](https://github.com/Microsoft/WinObjC/pull/1330#discussion_r86892725) [](ancestors = 86892725)